Sources/Views/AnimatableTableViewCell.swift

```diff
@@ -156,7 +159,9 @@ open var startPoint: GradientStartPoint = .top
   fileprivate func configureInspectableProperties() {
     configureAnimatableProperties()
     configureOpacity()
+    #if os(iOS)
```

Comment:
yes, you are right, we have to expose TableViewCellDesignable to the tvOS target. That's not good either.